### PR TITLE
Bug 1751979 - Use backfill projects and increased parallelism for shredder

### PR DESCRIPTION
**Pull request by relud**
[Bug 1751979](https://bugzilla.mozilla.org/show_bug.cgi?id=1751979)

### CODE REVIEW

File: dags/shredder.py [hunk 1]
```diff
@@ -95,6 +95,21 @@
...
+        "moz-fx-data-backfill-1",
+        "moz-fx-data-backfill-2",
+        "moz-fx-data-backfill-3",
+        "moz-fx-data-backfill-4",
+        "moz-fx-data-backfill-5",
+        "moz-fx-data-backfill-6",
+        "moz-fx-data-backfill-7",
+        "moz-fx-data-backfill-8",
+        "moz-fx-data-backfill-9",
+        "moz-fx-data-backfill-10",
```
[the remark this answers is not in the document]
Comment (relud) [reply dags/shredder.py]: updated to 20 through 29

[the remark this answers is not in the document]
Comment (whd) [reply dags/shredder.py]: I'm not sure how billing projects interact with actual resource permissions but airflow currently doesn't have access to any backfill projects (except `moz-fx-data-shredder`), perhaps something like:

https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-bigquery-batch/tf/prod/projects/main.tf

needs to be provisioned.

[the remark this answers is not in the document]
Comment (jasonthomas) [reply dags/shredder.py]: `billing_project` and `quota_project` are sometimes used interchangeably, but it ties back to which project the resource will be provisioned, in this case BigQuery create on-demand jobs, is going to run in. So @whd is right, the airflow service accounts needs to be able to create jobs against the project as well.

File: dags/shredder.py [hunk 2]
```diff
@@ -95,6 +95,21 @@
         "--parallelism=2",
         "--billing-project=moz-fx-data-bq-batch-prod",
         "--only=telemetry_derived.main_summary_v4",
+        "--no-use-dml",
+        # temporarily override parallelism and billing projects to use backfill projects
+        # to spread jobs out and increase speed of sanitizing columns for Bug 1751979
+        "--parallelism=10",
```
Comment (jklukas) [inline dags/shredder.py:101]: We have `         "--parallelism=2",` a few lines above this. We should make sure to remove that line.

**Review by jklukas — CHANGES_REQUESTED** [review]

File: dags/shredder.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jasonthomas) [reply dags/shredder.py]: > Should I file a DSRE ticket about giving Airflow SAs access to run query jobs in backfill projects?

whd beat you to it https://github.com/mozilla-services/cloudops-infra/pull/3804

**Pull request merged**